### ISSUE TITLE
end bracket (")") missing using double += on custom types.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1354]
### Expected

7

### Actual

Uncaught SyntaxError: missing ) after argument list

### Steps To Reproduce
[Live Bridge](http://live.bridge.net/#ee56cdd090b8d081c326664eb1d90e89)
[Dot Net Fiddle](https://dotnetfiddle.net/9rjA3J)
```csharp
/// <summary>
/// Creates a simple class
/// </summary>
public class App
{
    // Use the [Ready] attribute to automatically
    // run this method when the page is ready.
    [Ready]
    public static void Main()
    {
        IntWrapper @int = new IntWrapper(3);
        @int += @int += new IntWrapper(1);
	    	Console.WriteLine(@int.ToInt());
    }
}

public class IntWrapper
{
	public int value;
	
	public int ToInt ()
	{
		return value;
	}
	
    public IntWrapper (int value)
	{
		this.value = value;
	}
    
    public static IntWrapper operator + (IntWrapper a, IntWrapper b)
	{
		return new IntWrapper(a.value + b.value); 
	}
}
```

**Comment by LeonidVeriga** [comment]
Thanks for a great catch. We will try to fix this in 1.14

**Comment by calcock** [comment]
Greetings! 

You'll be surprised  about what I've found  on the web, just take a look <http://rdycushultu.truesoundstudio.net/lngox/46>

All best, calcock
